meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,350 @@
+        raw_admin = admin_message.get("raw")
+        if raw_admin is None or not hasattr(
+            raw_admin, "get_canned_message_module_messages_response"
+        ):
+            logger.warning(
+                "Unexpected canned-message response without raw message data"
+            )
+            return False
+        try:
+            canned_messages = raw_admin.get_canned_message_module_messages_response
+            self._cache_store.store_canned_message_fragment(canned_messages)
+            return True
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)